lib/pltr/v2/migrator/handleSpecialCases.js

```diff
@@ -41,9 +41,10 @@ export const handleMissingTemplatesFieldOnCards = (file) => {
 
   // Check, and leave the file completely un changed if there were no
   // problems, because that feels safer.
-  const thereAreMissingTemplates = file.cards.some(
-    (card) => !card.templates || !Array.isArray(card.templates)
-  )
+  const thereAreMissingTemplates =
+    file &&
+    file.card &&
```

Comment:
This looks like a typo(?)

Reply:
I think this caused the failing test.

Reply:
Oh yes of course 😅 thank you for checking ed